<launch>
    <arg name="weight_file"                 default="$(find dr_spaam_ros)/weight_file/ckpt_jrdb_pl_dr_spaam_phce_mixup_e20.pth"/>
    <group ns = "dr_spaam">
        <node pkg="multiple_sensor_person_tracking" type="dr_spaam_ros.py" name="dr_spaam_ros" output="screen">
            <param name="weight_file"               value="$(arg weight_file)" />
            <rosparam file="$(find sobit_follower)/param/dr_spaam_param.yaml" command="load" />
        </node>
    </group>
</launch>

